### PR TITLE
Converge to "vector" in lieu of list and set. Fixes erratum 6135.

### LINKED ISSUE #43: Erratum 6135: list, etc.
Errata ID: 6135
Status: Reported
Type: Editorial
Publication Format(s) : TEXT
Reported By: Ben Smyth
Date Reported: 2020-04-28

Section Global says:

list, series, set, and vector are seemingly used as synonyms. 

It should say:

Using list, series, set, xor vector would help at least one reader. 

Notes:

Additionally, consistent usage is desirable, e.g., page 31 uses "A list of extensions" whereas "A set of
extensions" is used on page 60. Elsewhere inconsistently usage causes confusion, e.g.,
Page 48:

client_shares: A list of offered KeyShareEntry values in descending
order of client preference.

This vector MAY be empty if the client is requesting a

(Replace "vector" with "list", or vice versa.)

### PR DESCRIPTION
**Pull request by chris-wood**
Closes #43.

### CODE REVIEW

**Comment by ekr** [comment]
Sorry, I should have weighed in on this one. I think we should converge to "list" in the text. "vector" is the name of type, and it's OK too. I think we should remove "set" and "sequence" Can you do that

**Comment by chris-wood** [comment]
List did seem more natural for the text, but the impression I got from Ben’s feedback was that he was expecting vector. I’ll swap these around.

**Comment by ekr** [comment]
Yeah, I can imagine that, but we're not bound by his opinion. Thx

File: draft-rescorla-tls-rfc8446-bis.md [hunk 1]
```diff
@@ -1632,7 +1633,7 @@ Here:
 - "extension_data" contains information specific to the particular
   extension type.
 
-The list of extension types is maintained by IANA as described in
+IANA maintains the extension type list as described in
```
Comment (ekr) [inline draft-rescorla-tls-rfc8446-bis.md:1636]: Let's revert this. I agree the phrasing is clumsy but I'd prefer not to make too any unneeded text changes.

**Pull request merged**